kover-gradle-plugin/src/main/kotlin/kotlinx/kover/gradle/plugin/tasks/reports/KoverHtmlTask.kt

```diff
@@ -28,7 +30,14 @@ internal abstract class KoverHtmlTask : AbstractKoverReportTask() {
     }
 
     fun printPath(): Boolean {
-        logger.lifecycle("Kover: HTML report for '$projectPath' file://${reportDir.get().asFile.canonicalPath}/index.html")
+        val clickablePath = URI(
```

Comment:
It looks like such code gives the same result:

```
        val clickablePath = URI(
            "file",
            "",
            file.resolve("index.html").canonicalPath,
            null
        ).toASCIIString()
```

Reply:
Perhaps it's even better to try like this:
```
        val clickablePath = URI(
            "file",
            "",
            reportDir.get().file("index.html").asFile.toURI().toASCIIString(),
            null
        ).toASCIIString()
```

Reply:
@shanshin I agree with the first one - the second one crashes for me, because the path ends up being something like `C;/path/to/file.html`, but `URI` requires a leading slash - `canonicalPath` is the bit that adds this slash in

Reply:
Actually, the first one appears to crash for me too 🤔

Reply:
```kotlin
reportDir.get().asFile.resolve("index.html").toURI().path
```
seems to work, as an alternative to
```kotlin
File(reportDir.get().asFile.canonicalPath, "index.html").toURI().path
```

Reply:
`reportDir.get().asFile.resolve("index.html").toURI().path`
Ok, this is also a good variant if it works stably on Windows

Reply:
Found some issues with that version on Windows :(

My recommendation would be to just use the original version in this PR - I took that straight from the Gradle codebase, so I assume that that's very battle-tested and reliable

Reply:
Ok. 
Could you please specify what problems have appeared? 
This may come in handy in the future in other places